webpack.config.babel.js

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+
+const STATIC_URL = process.env.STATIC_URL || 'http://localhost:5000/'
```

Comment:
@arnellebalane is there a way to make this respect django's static url settings? Note the ana is a django app the users will just add in installed apps and should just work out of the box just like django admin. We cannot have another server running the frontend, it should just be loaded by django's static server.

Reply:
@emanlodovice This build script is run before we publish the package to PyPI, therefore we have no way of knowing what the users' static urls are going to be without asking them for any settings. I just removed this config from the build script in a57d5bd and will just try to avoid any lazyloading as much as possible.